alerts_consumer.go

```diff
@@ -271,14 +285,21 @@ func (c *AlertsConsumer) SendBatch(batch [][]byte, tag string) error {
 		return kbc.PartialSendBatchError{ErrMessage: "failed to add events: " + err.Error(), FailedMessages: batch}
 	}
 
-	lg.TraceD("cloudwatch-add-datapoints", logger.M{"point-count": len(dats)})
-	_, err = c.cwAPI.PutMetricData(&cloudwatch.PutMetricDataInput{
-		Namespace:  aws.String(cloudwatchNamespace),
-		MetricData: dats,
-	})
-	// TODO: We could return that we failed to process this batch (as above) but for now just log
-	if err != nil {
-		lg.ErrorD("error-sending-to-cloudwatch", logger.M{"error": err.Error()})
+	if tag != "default" {
```

Comment:
Can we add a comment around this `default` special casing

Reply:
sure